### ISSUE TITLE
difference between baselineSERes18Conc and anatomy net

### ISSUE THREAD

**Issue by debbiely77** [wentaozhu/AnatomyNet-for-anatomical-segmentation#19]
Hi, thank you for sharing your nice work. 

I have some questions when running your code.
1. what's the  difference between baselineSERes18Conc and anatomy net? I check the two *.py file line by line, I thought the differences is only the relu function?  If so, is there such big gap between the performance of the two model that we must finetune anatomy net based on the pretrained baselineSERes18Conc model? 
2. Do we need the entire three database to reproduce your result? I used only the PDDCA dataset and I crop the image myself to exclude the non head neck area. The result is wierd like this. After 287 epoch the model converge, I use diceloss+0.5*Focalloss. And I use the Adam instead of your RMS+SGD since they don't work on my machine, and I don't know why.

epoch | train_loss | train_acc_BrainStem | train_acc_Chiasm | train_acc_OPL | train_acc_OPR | train_acc_Parotid_L | train_acc_Parotid_R
-- | -- | -- | -- | -- | -- | -- | --
287 | 0.159289687224056 | 0.832054673673114 | 0.368384247710526 | 0.440287856253575 | 0.432384147017327 | 0.793738396989537 | 0.797057225879442

val_loss | val_acc_BrainStem | val_acc_Chiasm | val_acc_OPL | val_acc_OPR | val_acc_Parotid_L | val_acc_Parotid_R
-- | -- | -- | -- | -- | -- | --
0.045590034552983 | 0.834816692158485 | 0.346497982045023 | 0.41390753766761 | 0.433888376484056 | 0.790529341529381 | 0.794535598142614

Currently, I'm not sure which step is run. I tried diceloss ,focal loss as well as three kind of other loss. The best dice coefficient for chiasm is less than 0.4. My training time is around 20 hours to get the model converged.

Would you please give me any suggestion? Thank you

**Comment by wentaozhu** [comment]
1 Finetune is always a better option.
2 Yes. PDDCA is too small. You can directly download my preprocessed data.
3 Yes. Basically, one day training should be enough.

**Comment by debbiely77** [comment]
Yes. I'm trying your entire dataset and it looks well since the loss decreases as it supposed to be. By the way, there is a lossweight for each organ in your code. How do you set them, manually or in any other way?
Sorry for so much questions.

**Comment by wentaozhu** [comment]
Please read the paper for the details.
AnatomyNet: Deep learning for fast and fully automated whole‐volume segmentation of head and neck anatomy https://www.biorxiv.org/content/biorxiv/early/2018/08/16/392969.full.pdf
Most read award in Medical Physics. Thank you!

**Comment by Faithyf** [comment]
@debbiely77  Can you share the PDDCA dataset ?Thank you
